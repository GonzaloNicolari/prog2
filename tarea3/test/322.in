# prueba de insertar_en_binario, mayor , remover_mayor, remover_de_binario, imprimir_binario
es_vacio_binario
mayor
remover_mayor
remover_de_binario l
insertar_en_binario (10,l)
es_vacio_binario
imprimir_binario
mayor
remover_mayor
es_vacio_binario
imprimir_binario
remover_de_binario l
insertar_en_binario (10,l)
es_vacio_binario
imprimir_binario
mayor
remover_de_binario l
es_vacio_binario
imprimir_binario
insertar_en_binario (10,l)
insertar_en_binario (8,g)
mayor
insertar_en_binario (13,o)
mayor
remover_mayor
imprimir_binario
mayor
insertar_en_binario (9,h)
insertar_en_binario (6,e)
mayor
remover_de_binario g
imprimir_binario
insertar_en_binario (4,b)
insertar_en_binario (12,n)
mayor
insertar_en_binario (15,q)
insertar_en_binario (7,f)
remover_de_binario l
imprimir_binario
remover_de_binario h
imprimir_binario
remover_de_binario e
remover_de_binario b
imprimir_binario
remover_de_binario n
imprimir_binario
remover_de_binario f
imprimir_binario
remover_mayor
Fin
