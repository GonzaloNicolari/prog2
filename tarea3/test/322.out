1>#  prueba de insertar_en_binario, mayor , remover_mayor, remover_de_binario, imprimir_binario.
2>Vacio.
3>b es vacío.
4>b es vacío.
5>No estaba la frase en 'b'.
6>Insertado.
7>No vacío.
8>
(10,l)
9>(10,l)
10>Removido el mayor.
11>Vacio.
12>
13>No estaba la frase en 'b'.
14>Insertado.
15>No vacío.
16>
(10,l)
17>(10,l)
18>Removido.
19>Vacio.
20>
21>Insertado.
22>Insertado.
23>(10,l)
24>Insertado.
25>(13,o)
26>Removido el mayor.
27>
(10,l)
-(8,g)
28>(10,l)
29>Insertado.
30>Insertado.
31>(10,l)
32>Removido.
33>
(10,l)
--(9,h)
-(6,e)
34>Insertado.
35>Insertado.
36>(12,n)
37>Insertado.
38>Insertado.
39>Removido.
40>
--(15,q)
-(12,n)
(9,h)
--(7,f)
-(6,e)
--(4,b)
41>Removido.
42>
--(15,q)
-(12,n)
(7,f)
-(6,e)
--(4,b)
43>Removido.
44>Removido.
45>
--(15,q)
-(12,n)
(7,f)
46>Removido.
47>
-(15,q)
(7,f)
48>Removido.
49>
(15,q)
50>Removido el mayor.
51>Fin.
